frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar/tests/common.unit.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import userEvent from "@testing-library/user-event";
+import {
+  createMockCard,
+  createMockModerationReview,
+} from "metabase-types/api/mocks";
+import { screen } from "__support__/ui";
+import { setup } from "./setup";
+
+describe("QuestionInfoSidebar", () => {
+  describe("description", () => {
+    it.each([
+      createMockCard({ name: "Question", description: "abc", dataset: false }),
+      createMockCard({ name: "Model", description: "abc", dataset: true }),
+    ])("should display description of a $card.name", async card => {
+      await setup({ card });
+      expect(screen.getByText(card.description ?? "")).toBeInTheDocument();
+    });
+
+    it("should not allow to add the description without write permissions", async () => {
+      const card = createMockCard({ description: null, can_write: false });
+      await setup({ card });
+
+      expect(screen.getByPlaceholderText("No description")).toBeDisabled();
+    });
+
+    it("should not allow to edit the description without write permissions", async () => {
+      const card = createMockCard({ description: "abc", can_write: false });
+      await setup({ card });
+
+      // show input
+      userEvent.click(screen.getByTestId("editable-text"));
+
+      const input = screen.getByPlaceholderText("Add description");
+      expect(input).toHaveValue(card.description);
+      expect(input).toBeDisabled();
+    });
+  });
+
+  describe("model detail link", () => {
+    it("is shown for models", async () => {
+      const card = createMockCard({ name: "abc", dataset: true });
+      await setup({ card });
+
+      const link = screen.getByText("Model details");
+
+      expect(link).toBeInTheDocument();
+      expect(link).toHaveAttribute(
+        "href",
+        `/model/${card.id}-${card.name}/detail`,
```

Comment:
decided not to use `Urls` here because getting a url from a raw card (not a question) should be deprecated soon

Reply:
From your comment, I get the impression that the previous tests use `Urls` which expects a raw card, but it actually already expects a question. What if we create a `Question` from this `card` and pass to `Urls` instead?

Reply:
I've updated the test to use `Urls` however the method uses the raw `Card` object

Reply:
I was referring to the method used in the deleted test file
https://github.com/metabase/metabase/blob/d25e808aa5ff8ad2568ecf8bc929ed2e6254195d/frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar.unit.spec.js#L165

This one expects a question instance.
https://github.com/metabase/metabase/blob/3b80635e56f0736eec1bd86b0f30d9f71d9ab8f6/frontend/src/metabase-lib/urls.ts#L25-L34

Though, I just realize we now have 2 places for managing urls.
1. `metabase-lib/urls.ts`
2. `metabase/lib/urls/*`

I believe the first one is only used within the metabase-lib itself since it's not allowed to reference code outside it directory. But to engineers, it could be confusing which one should be used.